### PR TITLE
Single cell aligner and 10x

### PR DESCRIPTION
**Pull request by Alex-Davydov**

### CODE REVIEW

File: src/main/resources/presets/blocks/01-align.yaml [hunk 1]
```diff
@@ -426,7 +426,6 @@ align-single-cell:
   abstract: true
   flags:
     - species
-    - materialType
```
Comment (dbolotin) [inline src/main/resources/presets/blocks/01-align.yaml]: Why?

File: src/main/resources/presets/blocks/01-align.yaml [hunk 2]
```diff
@@ -446,27 +445,27 @@ align-single-cell:
...
-          mapperMaxSeedsDistance: 15
+          mapperAbsoluteMinScore: 250
+          mapperRelativeMinScore: 0.75
+          mapperMinSeedsDistance: 6
+          mapperMaxSeedsDistance: 6
           alignmentStopPenalty: 0
           absoluteMinScore: 150
           relativeMinScore: 0.8
-          maxHits: 7
+          maxHits: 9
```
Comment (dbolotin) [inline src/main/resources/presets/blocks/01-align.yaml]: Really needed?

File: src/main/resources/presets/blocks/01-align.yaml [hunk 3]
```diff
@@ -489,27 +488,27 @@ align-single-cell:
...
-          mapperMaxSeedsDistance: 10
+          mapperAbsoluteMinScore: 100
+          mapperRelativeMinScore: 0.8
+          mapperMinSeedsDistance: 5
+          mapperMaxSeedsDistance: 5
           alignmentStopPenalty: 0
           absoluteMinScore: 150
           relativeMinScore: 0.8
-          maxHits: 10
+          maxHits: 3
```
Comment (dbolotin) [inline src/main/resources/presets/blocks/01-align.yaml:512]: ? again, strange difference from V params...

File: src/main/resources/presets/protocols/10x.yaml [hunk 4]
```diff
@@ -86,154 +77,56 @@
...
-                  lower: 3.0
+                  lower: 150.0
   assemblePartial:
     inheritFrom: assemblePartial-base
   assemble:
     inheritFrom: assemble-with-consensus-base
     overrides:
       clnaOutput: true
+      cellLevel: true
+      consensusAssemblerParameters:
```
Comment (dbolotin) [inline src/main/resources/presets/protocols/10x.yaml:107]: Maybe move whole block to `assemble-with-consensus-base` ?

File: src/main/resources/presets/protocols/10x.yaml [hunk 5]
```diff
@@ -247,71 +140,60 @@
               predicates:
                 - metrics:
                     - type: group_metric_nunique
+                      reportHist: null
                       keys:
                         - allTags:Molecule
                   operator:
                     type: group_operator_cumtop
                     share: 0.95
+                    accept: High
```
Comment (dbolotin) [inline src/main/resources/presets/protocols/10x.yaml]: This is default value, can be removed.

**Comment by dbolotin** [comment]
Also waiting for  #1064

File: src/main/resources/presets/protocols/10x.yaml (hunk 4 above)
Comment (Alex-Davydov) [reply src/main/resources/presets/protocols/10x.yaml:107]: Other single cell protocols, like BD, also use `assemble-with-consensus-base`, but not at cell level

File: src/main/resources/presets/protocols/10x.yaml [hunk 6]
```diff
@@ -86,156 +79,57 @@
...
                   - type: group_metric_kmer_diversity
-                    sequences: [ "targets" ]
+                    sequences:
+                      - targets
                     k: 12
                     minimalQuality: 20
                     topShare: 0.7
                     minSequenceCount: 4
                     reportHist:
+                      log: false
```
Comment (dbolotin) [inline src/main/resources/presets/protocols/10x.yaml]: Not needed.

File: src/main/resources/presets/protocols/10x.yaml [hunk 7]
```diff
@@ -86,156 +79,57 @@
...
                       binNumber: 40
+                      minBinWidth: 0.0
+                      multiplyWeightByKey: false
```
Comment (dbolotin) [inline src/main/resources/presets/protocols/10x.yaml]: Also delete.

**Review by dbolotin — CHANGES_REQUESTED** [review]
Final polishing required.

**Review by dbolotin — APPROVED** [review]

**Pull request merged**